app/oxalis/thirdparty/BrainTracing.scala

```diff
@@ -78,11 +78,11 @@ object BrainTracing extends LazyLogging with FoxImplicits {
     result.contains("ist derzeit nicht verf&uuml;gbar.")
 
   private def signalOverTime(
-    time: Long,
-    project: Box[Project],
-    task: Box[Task],
-    annotation: Option[Annotation],
-    user: User) = {
+                              time: Long,
```

Comment:
Why this much indentation? :straight_ruler:

Reply:
IntelliJ did that automatically, I assumed it probably has a good reason…? :D